eng/WpfArcadeSdk/tools/ApiCompat.targets

```diff
@@ -10,8 +10,9 @@
                                           and !($(MSBuildProjectName.Contains('_wpftmp')))">true</RunNetFrameworkApiCompat>
   </PropertyGroup>
 
-  <!-- Only enable Ref API Compat for the lib projects using hand-crafted reference assemblies -->
-  <PropertyGroup Condition="'$(RunRefApiCompat)'==''">
+  <!-- Only enable Ref API Compat for the lib projects using hand-crafted reference assemblies,
+  but not if we're generating the reference assemblies -->
```

Comment:
nit: $(GenerateReferenceAssemblySource)'!='true' for when it is explicitly false.